### PR TITLE
7z, 7za, 7zr, aapt: add pt_PT translations

### PR DESCRIPTION
**Pull request by MicaelMCarvalho**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by MicaelMCarvalho** [comment]
Waiting for review from @waldyrious. Every other check done and green

File: pages.pt_PT/common/aapt.md
Comment (kbdharun) [inline pages.pt_PT/common/aapt.md:9]: Thanks for your contribution, can you update the path placeholders in this page too.

Comment (MicaelMCarvalho) [reply pages.pt_PT/common/aapt.md:9]: Done.

**Review by kbdharun — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**
